diablo/externals/kaltura.py

```diff
@@ -503,7 +503,16 @@ def _event_to_json(event):
                 }
             return api_json
 
-    return [_event_to_json(event) for event in scheduled_events]
+    raw_list = [_event_to_json(event) for event in scheduled_events]
+    if organize_results:
+        recurring_events = list(filter(lambda e: e['recurrenceType'] == 'Recurring', raw_list))
+        standalone_events = list(filter(lambda e: e['recurrenceType'] not in ['Recurrence', 'Recurring'], raw_list))
```

Comment:
@johncrossman Is this right? Recurring events have type 'Recurring', and standalone events are all other events except those with type 'Recurrence,' which we ignore?

Reply:
@paulkerschen I reverted to what was there before... And what was there before was composed when I was new to the Kaltura API.  I'll have a closer look tomorrow and decide if improvements are needed. Thanks for asking!